### PR TITLE
single file component: PlayerStatus

### PR DESCRIPTION
**Pull request by bafolts**

### CODE REVIEW

File: src/components/overview/PlayerInfo.ts [hunk 1]
```diff
@@ -73,7 +70,7 @@ export const PlayerInfo = Vue.component('player-info', {
       }
       for (let i = 0; i < hiddenPlayersIndexes.length; i++) {
         if (hiddenPlayersIndexes.includes(i)) {
-          hidePlayerData(this.$root as unknown as typeof mainAppSettings.methods, i);
+          (this.$root as unknown as typeof mainAppSettings.methods).setVisibilityState('pinned_player_' + i, false);
```
Comment (kberg) [inline src/components/overview/PlayerInfo.ts:73]: I see why you are reintroducing duplication. Can you just add a TODO to find a way to deal with common code?

**Review by kberg — APPROVED** [review]
Minor request, otherwise lgtm

File: src/components/overview/PlayerInfo.ts (hunk 1 above)
Comment (bafolts) [reply src/components/overview/PlayerInfo.ts:74]: Need to see if the single file component pattern will allow us to gain type safety with `this.$root`.

**Comment by kberg** [comment]
It doesn't look as if this test was run. :'-(

**Comment by bafolts** [comment]
> It doesn't look as if this test was run. :'-(

The checks show the tests run.

**Comment by kberg** [comment]
This specific one? Because it wasn't passing. Help me understand, am I
missing something?

>

**Comment by bafolts** [comment]
Ah I was missing something. I didn't realize this test wasn't running. As my generation would say, "my bad".

**Comment by kberg** [comment]
My generation says it now, too. :b

**Pull request merged**